planner/core/testdata/plan_suite_out.json

```diff
@@ -1420,6 +1420,94 @@
       }
     ]
   },
+  {
+    "Name": "TestPushdownDistinctEnable",
+    "Cases": [
+      {
+        "SQL": "select /*+ HASH_AGG() */ a, count(distinct a) from t;",
+        "Plan": [
+          "Projection_4 1.00 root  test.t.a, Column#5",
+          "└─HashAgg_8 1.00 root  funcs:count(distinct test.t.a)->Column#5, funcs:firstrow(Column#6)->test.t.a",
+          "  └─TableReader_9 1.00 root  data:HashAgg_5",
+          "    └─HashAgg_5 1.00 cop[tikv]  group by:test.t.a, funcs:firstrow(test.t.a)->Column#6",
+          "      └─TableFullScan_7 10000.00 cop[tikv] table:t keep order:false, stats:pseudo"
+        ],
+        "Result": [
+          "1 2"
```

Comment:
the result is unstable because of `first_row(a)` and it has two distinct values.

Reply:
Thanks. Added two settings in variables, PTAL
```
+       tk.MustExec(fmt.Sprintf("set session %s=1", variable.TiDBHashAggPartialConcurrency))
+       tk.MustExec(fmt.Sprintf("set session %s=1", variable.TiDBHashAggFinalConcurrency))
```